### DISCUSSION TITLE
Change the homepage html

### DISCUSSION THREAD

**Discussion by cs-mshah** [HEIGE-PCloud/DoIt#1413] [Q&A]
I am interested in changing the home page template but want to keep the same header. To be specific, this is my [blog homepage](https://cs-mshah.github.io/) and this is the HTML page that I want to create instead of the default one from DoIt:
https://rishubhpar.github.io/

Can you support this kind of academic homepage template natively? This would be highly useful for researchers who maintain a homepage and a blog.

I tried following these:
https://gohugo.io/templates/home/
https://timhilliard.com/blog/static-home-page-in-hugo/

however the home HTML was not changed. I would highly appreciate any help.

**Comment by CXwudi** [comment]
I think you would have to modify the [`layouts/index.html`](https://github.com/HEIGE-PCloud/DoIt/blob/main/layouts/index.html) template to archive different homepage, but I am too sure

**Comment by cs-mshah** [comment]
Thanks for the suggestions. I made changes in the `index.html` and `profile.html` on the blog side. I was able to achieve the changes that I wanted.
